shell.nix

```diff
@@ -18,6 +18,11 @@
     delve
     golangci-lint
     gotools
+
+    kube3d
+    kubectl
+    k9s
+    kubernetes-helm
```

Comment:
The kubernetes toolchain should also be added to asdf's .tool-versions

Extracting it as a GH Issue for someone else to take over.

Reply:
https://github.com/smartcontractkit/chainlink-starknet/issues/70